### ISSUE TITLE
svg::Group::getBoundingBox() is empty

### ISSUE THREAD

**Issue by sansumbrella** [cinder/Cinder#843]
When analyzing a group of lines, the group bounding box is incorrect.

```svg
<g id="position">
  <line fill="none" stroke="#ED1C24" stroke-miterlimit="10" x1="246.6" y1="284" x2="246.6" y2="316"/>
  <line fill="none" stroke="#ED1C24" stroke-miterlimit="10" x1="262.6" y1="300" x2="230.6" y2="300"/>
</g>
```

The following Cinder code looks at that group and should return its center:
```c++
vec2 getCenter( svg::Group *group ) {
  return group->getBoundingBoxAbsolute()->getCenter();
}
```

Expected result:
The code returns `vec2( 246.6, 300 )`

Actual result:
The code returns `vec2(0)`

**Comment by sansumbrella** [comment]
As a temporary workaround, the following produces the expected results:
```c++
vec2 getCenter( svg::Group *group ) {
  auto shape = group->getShape();
  return shape->calcBoundingBox()->getCenter();
}
```

Unfortunately, it requires a trip through Shape creation.

**Comment by andrewfb** [comment]
Thanks - should be fixed by 1d5c673f860d236813ffadf7b9862d50bff093ba
